packages/client-beta/src/apps/Soundboard/Soundboard.tsx

```diff
@@ -25,9 +28,34 @@ function Soundboard() {
   createEffect(() => {
     fetch('/api/x-sounds')
       .then((res) => res.json())
-      .then((res) => setXsounds(res.sounds));
+      .then((res) => {
+        setXsounds(res.sounds);
+        setFilteredXSounds(res.sounds);
+      });
+
+    fetch('/api/x-sounds/tags')
```

Comment:
I feel like fetching should be done in some kind of client, so that in components we don't call `fetch` and don't have to worry about JSON parsing - just calling a method like `getXSoundTags()` that returns a Promise with a list of tags.